modules/core/src/core/animation-loop.js

```diff
@@ -213,7 +218,7 @@ export default class AnimationLoop {
     this.onRender(this.animationProps);
     // end callback
 
-    if (this.offScreen) {
+    if (this.offScreen && this.gl.commit) {
```

Comment:
Who sets this `this.gl.commit`, may be add a comment?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/commit

Calling `gl.commit` is required in FF but not in Chrome 70.